app/src/main/kotlin/com/bitwarden/authenticator/data/platform/datasource/network/core/ResultCall.kt

```diff
@@ -0,0 +1,95 @@
+package com.bitwarden.authenticator.data.platform.datasource.network.core
+
+import com.bitwarden.authenticator.data.platform.util.asFailure
+import com.bitwarden.authenticator.data.platform.util.asSuccess
+import okhttp3.Request
+import okio.IOException
+import okio.Timeout
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.HttpException
+import retrofit2.Response
+import java.lang.reflect.Type
+
+/**
+ * The integer code value for a "No Content" response.
+ */
+private const val NO_CONTENT_RESPONSE_CODE: Int = 204
+
+/**
+ * A [Call] for wrapping a network request into a [Result].
+ */
+@Suppress("TooManyFunctions")
+class ResultCall<T>(
```

Comment:
Can we annotate with `OmitFromCoverage`? I don't see tests for this in the PM app either.

Reply:
ye for sure maybe we should do it the main app as well